src/docfx/lib/HtmlUtility.cs

```diff
@@ -16,25 +15,27 @@ internal static class HtmlUtility
         private static readonly Func<HtmlAgilityPack.HtmlAttribute, int> s_getValueStartIndex =
             ReflectionUtility.CreateInstanceFieldGetter<HtmlAgilityPack.HtmlAttribute, int>("_valuestartindex");
 
-        public static HtmlNode LoadHtml(string html)
-        {
-            var doc = new HtmlDocument();
-            doc.LoadHtml(html);
-            return doc.DocumentNode;
-        }
+        public static string HtmlPostProcess(string html, CultureInfo culture) => HtmlPostProcess(LoadHtml(html), culture);
 
-        public static string TransformHtml(string html, Func<HtmlNode, HtmlNode> transform)
+        public static string HtmlPostProcess(HtmlNode html, CultureInfo culture)
         {
-            HtmlDocument document = new HtmlDocument();
-            document.LoadHtml(html);
-            return transform(document.DocumentNode).OuterHtml;
+            html = html.StripTags();
+            html = html.AddLinkType(culture.Name.ToLowerInvariant())
```

Comment:
so the culture name would always be appended to the resolved URL. We have some logic of appending the locale in the compare tool, we can remove them then.

Reply:
It will only append if it is an absolute path (like before), it does not affect the compare tool.

Reply:
https://apidrop.visualstudio.com/Content%20CI/_git/docfx-v3-migration?path=%2Fscripts%2Fmigration%2Fcompare%2Fnormalize-html.js&version=GBmaster&line=65&lineStyle=plain&lineEnd=66&lineStartColumn=1&lineEndColumn=1